### PR TITLE
Add: getDomPathString function

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy

### CODE REVIEW

File: src/utils/getDomPathString.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const getDomPathString = function (domPath) {
```
Comment (ironsidevsquincy) [inline src/utils/getDomPathString.js]: I'd give this the same api as `getDomPath`, i.e. takes an element

**Comment by ironsidevsquincy** [comment]
Test?

File: tests/browser/specs/utils/getDomPathStringSpec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	});
+
+	it("Display the trackable path of a DOM element in outer to inner order in string format", function () {
+		rootEl.innerHTML += '<section data-trackable="c">' +
+								'<span data-trackable="b">' +
+									'<div>' +
+										'<button data-trackable="a" id="d"></button>' +
+									'</div>' +
+								'</span>' +
+							'</section>';
```
Comment (ironsidevsquincy) [inline tests/browser/specs/utils/getDomPathStringSpec.js]: odd indentation here. might be easier to use use backticks as strings can span newlines

File: tests/browser/specs/utils/getDomPathStringSpec.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/*global describe, it, expect, beforeEach*/
+/*jshint expr: true */
+const getDomPathString = require('../../../../src/utils/getDomPathString');
+
+describe("getDomPathString", function () {
+
+	const rootEl = document.body;
```
Comment (ironsidevsquincy) [inline tests/browser/specs/utils/getDomPathStringSpec.js]: probably fine, but wouldn't overwrite the body, just add a child to it to use

File: src/utils/getDomPathString.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import getDomPath from './getDomPath';
```
Comment (ironsidevsquincy) [inline src/utils/getDomPathString.js]: everything else is using commonJS requires, so best to stick to that

File: src/utils/getDomPathString.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+
+const getDomPathString = function (element) {
+	return getDomPath(element, []).reverse().join(" | ");
```
Comment (ironsidevsquincy) [inline src/utils/getDomPathString.js]: single quotes

File: tests/browser/specs/analytics/ctaSpec.js [hunk 6]
```diff
@@ -21,7 +21,7 @@ describe("CTA", function () {
 		rootEl.addEventListener('oTracking.event', function listener(e) {
 			expect(e.detail.nodeName).to.equal('a');
 			expect(e.detail.domPath).to.equal('abc | xyz');
-			expect(e.detail.domPathTokens).to.eql(['abc', 'xyz']);
+			expect(e.detail.domPathTokens).to.eql(['xyz', 'abc']);
```
Comment (ironsidevsquincy) [inline tests/browser/specs/analytics/ctaSpec.js]: was this broken then?

File: tests/browser/specs/utils/getDomPathStringSpec.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+	let div;
+
+	beforeEach(function () {
+		div = document.createElement('div');
+		document.body.appendChild(div);
+	});
+
+	afterEach(function () {
+		document.body.innerHTML = null;
```
Comment (ironsidevsquincy) [inline tests/browser/specs/utils/getDomPathStringSpec.js]: anal here, but best practice to tidy up just what you do, i.e. use `removeChild` here to remove the div

**Comment by ironsidevsquincy** [comment]
bar comment, :+1:

File: tests/browser/specs/analytics/ctaSpec.js (hunk 6 above)
Comment (andygout) [inline tests/browser/specs/analytics/ctaSpec.js]: Yup - hence why when turned to a string it requires `reverse()` (and did prior to now as well).

Comment (ironsidevsquincy) [inline tests/browser/specs/analytics/ctaSpec.js]: But the tests weren't failing before, were they?

https://circleci.com/gh/Financial-Times/n-instrumentation/86

How were the passing

**Pull request merged**